analysis.js

```diff
@@ -72,19 +56,19 @@ async function startAnalysis(context, scope) {
     .catch((error) => {
       // Send the validation to the device.
       // That way we create an error in the dashboard for feedback.
-      dashboard_device.sendData({ variable: "validation", value: `Error when creating the device ${error}`, metadata: { color: "red" } });
+      Resources.devices.sendDeviceData(scope[0].device, {
+        variable: "validation",
+        value: `Error when creating the device ${error}`,
+        metadata: { color: "red" },
+      });
       throw error;
     });
 
   // To add Configuration Parameters to the device:
-  account.devices.paramSet(result.device_id, { key: "param_key", value: "10", sent: false });
-
-  // To add any data to the device that was just created:
-  // const device = new Device({ token: result.token });
-  // device.sendData({ variable: 'temperature', value: 17 });
+  Resources.devices.paramSet(result.device_id, { key: "param_key", value: "10", sent: false });
 
   // Send feedback to the dashboard:
-  dashboard_device.sendData({ variable: "validation", value: "Device succesfully created!", metadata: { type: "success" } });
+  Resources.devices.sendDeviceData(scope[0].device, { variable: "validation", value: "Device succesfully created!", metadata: { type: "success" } });
```

Comment:
Do not use scope[0].device as parameter. Prefer to have it on a variable, so user can better understand the code.

const deviceID = scope[0].device;